code/modules/religion/religion_structures.dm

```diff
@@ -43,6 +49,7 @@
 			icon = sect_to_altar.altar_icon
 		if(sect_to_altar.altar_icon_state)
 			icon_state = sect_to_altar.altar_icon_state
+	update_overlays() //Light the candles!
```

Comment:
This just returns a list and does nothing to update the overlays

Reply:
You want `update_appearance` here

Reply:
I stored the candle flame for the non-standard altars as an overlay to save on space and time, you're sure?

Reply:
i'm a big dumb baby that doesn't know how this code works lmao